### ISSUE TITLE
Remove "Camera Roll" album view

### ISSUE THREAD

**Issue by joewalnes** [novaphotos/nova-ios-app#63]
Now that we only ever show photos from the camera roll, the album view screen is redundant and can be removed.

**Comment by pix0r** [comment]
When you say "album view screen" do you mean the UIImagePickerController that shows a grid view of photo thumbnails?  So we'd essentially remove the button from the image view that exposes this screen, and the user will use swipe navigation to bring up older photos?

**Comment by joewalnes** [comment]
Yep, the UIImagePickerController. Leave the button in, but have it jump directly to the last stored photo without the showing the grid in between.

**Comment by joewalnes** [comment]
Just realized which button you were talking about. Yes, remove it.
